src/trans/gpu/internal/trmtol_pack_unpack.F90

```diff
@@ -251,16 +258,17 @@ SUBROUTINE TRMTOL_UNPACK(ALLOCATOR,HTRMTOL_UNPACK,FOUBUF,PREEL_COMPLEX,KF_CURREN
 #endif
 
 OFFSET_VAR=D_NPTRLS(MYSETW)
+ILOEN_MAX=MAXVAL(G_NLOEN)
 #ifdef OMPGPU
 #endif
 #ifdef ACCGPU
 !$ACC PARALLEL LOOP PRIVATE(IGLG,IOFF_LAT,ISTA,RET_REAL,RET_COMPLEX) FIRSTPRIVATE(KF_CURRENT,&
-!$ACC&              KF_TOTAL,OFFSET_VAR,G_NLOEN_MAX) DEFAULT(NONE) &
+!$ACC&              KF_TOTAL,OFFSET_VAR) DEFAULT(NONE) &
 !$ACC&              ASYNC(1) TILE(32,16,1)
 #endif
 DO KGL=1,D_NDGL_FS
   DO JF=1,KF_CURRENT
-    DO JM=0,G_NLOEN_MAX/2
+    DO JM=0,ILOEN_MAX/2
```

Comment:
Same thing here as above. I'm concerned we might break things on AMD if we start missing out variables when `DEFAULT(NONE)` is specified.